config/locales/en.yml

```diff
@@ -55,4 +56,15 @@ en:
       h2_1: "sample application."
       h3: "Micropost Feed"
       signup: "Sign up now!"
-      rails_logo: "Rails logo"
\ No newline at end of file
+      rails_logo: "Rails logo"
+
+  users:
+    new: 
+      h1: "Sign up"
+      create: "Create my account"
+  shared:
+    _error_messages:
```

Comment:
No need to use _  at the start of a variable name. Start with a lowercase letter.

Reply:
Ok a, I'll remember it